src/store/utils.js

```diff
@@ -50,3 +50,10 @@ export function downloadFileInBrowser(filename, content, type) {
         document.body.removeChild(link);
     }
 }
+
+export function stxtap(el, func) {
+    if (el && !el.safeClickTouchEvents) {
```

Comment:
@aminroosta fixed. `safeClickTouchEvents` is appended to the element via `CIQ.safeClickTouch`, which is called in `CIQ.installTapEvent`